components/compliance-service/ingest/pipeline/processor/reports_project_tagger.go

```diff
@@ -23,6 +23,10 @@ func BundleReportProjectTagger(authzClient iam_v2.ProjectsClient) message.Compli
 // currently in the queue. The 'bundleSize' is the number of messages that can use the current project rules from authz.
 // When the bundleSize zero or less we need to refetch the project rules.
 func reportProjectTagger(in <-chan message.Compliance, authzClient iam_v2.ProjectsClient) <-chan message.Compliance {
+	if authzClient == nil {
+		logrus.Error("no authz client found for project tagging; skipping project tagging")
```

Comment:
we should never have a nil authzClient here in a working env -- bc we would have failed during startup if we couldnt get that connection. this is basically here bc the ruby compliance scanner tests will fail without it